default/python/universe_generation/galaxy.py

```diff
@@ -109,15 +422,13 @@ def spiral_galaxy_calc_positions(positions, arms, size, width):
             break
 
         if not attempts:
-            print "Spiral galaxy shape: giving up on placing star", i,\
-                  ", can't find position sufficiently far from other systems"
+            print "Spiral galaxy shape: giving up on placing star {}, can't find position sufficiently far from other systems.".format(i)
```

Comment:
Changing the print statement to use the new recommended formatting is fine, but why did you remove the line break? We're trying to conform to PEP8, and this results in a slightly too long line (and the notorious PEP8 warning).